nbs/explains/minidataapi.ipynb

```diff
@@ -0,0 +1,516 @@
+{
```

Comment:
_Commented on notebook `nbs/explains/minidataapi.ipynb` Cell 9 Line 3_
```markdown
## .insert()

Add a new record to the database. Must accept Python dictionaries, dataclasses, and standard classes. Returns an instance of the new record.
```
Have you checked whether fastsql supports dicts? I can't recall if it does. If there's no tests of that in fastsql, but it does work, please add tests/examples.

Reply:
fastsql doesn't support the use of adding records via dicts. I'll add that capability in that library.